src/components/tabs/tabs.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import { Tab, Tabs } from './tabs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+
+storiesOf('Tabs', module).add(
+  'default',
+  () => {
+    const selectOption = ['top', 'left'];
+    const align = select('align', OptionsHelper.alignBinary, OptionsHelper.alignBinary[0]);
+    const position = select('position', selectOption, selectOption[0]);
```

Comment:
Can we take the default value from the component? I.e. `Tabs.defaultProps.position`